conf/spgwu.bess

```diff
@@ -16,14 +16,28 @@ from conf.utils import *
 conf_file = get_env('CONF_FILE', 'conf/spgwu.json')
 conf = get_json_conf(conf_file, False)
 
-# Maximum number of flows to manage ip frags for re-assembly
+# Maximum number of flows to manage ip4 frags for re-assembly
 max_ip_defrag_flows = None
 try:
     max_ip_defrag_flows = int(conf["max_ip_defrag_flows"])
 except ValueError:
-    print('Invalid value for max_ip_defrag_flows. Not installing IPDefrag module.')
+    print('Invalid value for max_ip_defrag_flows. Not installing IP4Defrag module.')
 except KeyError:
-    print('max_ip_defrag_flows value not set. Not installing IPDefrag module.')
+    print('max_ip_defrag_flows value not set. Not installing IP4Defrag module.')
+
+# Enable ip4 fragmentation
+ip_frag_with_eth_mtu = None
```

Comment:
you can use blocks to define customizations like we do for per interface configs. 

```json
"ipdefrag": {
               "flows": 1000,
               "mtu": 1400,
},
```

Reply:
Should mtu be moved to interface specific config?